tool/generator/bcd.dart

```diff
@@ -13,6 +13,13 @@ import 'filesystem_api.dart';
 /// property status (standards track, experimental, deprecated) and supported
 /// browser (chrome, safari, firefox) info.
 class BrowserCompatData {
+  static final _eventHandlers = <String, Set<BCDPropertyStatus>>{};
```

Comment:
Switch to put the type on the left for members?

Reply:
https://dart.dev/effective-dart/design#do-type-annotate-fields-and-top-level-variables-if-the-type-isnt-obvious

We're a bit inconsistent here with this practice between files. In `translator.dart` we prefer omitting the type if it's obvious (like with collections), but in `doc_provider.dart`, we always keep the types to the left. I don't have a strong preference here and these are the only two fields in this file that would be affected, so done.